tracker/src/plausible.js

```diff
@@ -173,7 +173,7 @@
       window.addEventListener('phx:navigate', info => trigger('pageview', {u: info.detail.href}));
     {{/unless}}
 
-    ['phx-click', 'phx-change', 'phx-submit', 'phx-viewport-top', 'phx-viewport-bottom', 'phx-mounted', 'phx-connected', 'phx-disconnected'].map((name) => {
```

Comment:
Have you regression tested current live view features? phx-click and events?

Reply:
yes.